packages/next/taskfile.js

```diff
@@ -155,7 +155,10 @@ export async function ncc_babel_bundle_packages(task, opts) {
   await task
     .source(opts.src || 'bundles/babel/packages-bundle.js')
     .ncc({
-      packageName: `@babel/core`,
+      // note: this packageName can not be the name of
+      // an externals package or else it will not be
+      // externalized by ncc
+      packageName: `@babel/preset-env`,
```

Comment:
Maybe we should give a name that won't collide with anything else? `@next/babel-bundle`?

Reply:
It looks like we don't need this at all, I removed `packageName` and `bundleName` since it's more correct as it's a bundle of packages and not a single package.